src/gesture.js

```diff
@@ -323,9 +323,8 @@ export class Gestures {
    * recognizer to proceed.
    * @param {!GestureRecognizer} recognizer
    * @param {number} offset
-   * @private
    */
-  signalReady_(recognizer, offset) {
+  signalReady(recognizer, offset) {
```

Comment:
Shouldn't the name be changed at the external call sites too?

Reply:
https://github.com/ampproject/amphtml/pull/14905/files#diff-971cca60811e44231f07a088cc5bc30fR584

Reply:
Aha, external to the class, internal to the file.

Reply:
What do you think about defining `@friend` that would expose these to others in the file? It wouldn't help for truly external callers, but some of these classes really are tied closely.

Reply:
Would this work? http://usejsdoc.org/tags-protected.html